packages/client/__tests__/pages/ResourceIndex.spec.js

```diff
@@ -59,13 +71,103 @@ describe('Test the resource index page', () => {
         expect(asFragment()).toMatchSnapshot()
     })
     test('should load posts data and display on table', () => {
-        const { debug } = render(
+        render(
             <Router history={history}>
                 <ResourceIndex {...props} />
             </Router>
         )
-
         expect(window.Flamingo.request.get).toHaveBeenCalled()
     })
-    test('should be able to edit a post from the post table', () => {})
+    test('per page button changes the pers page value for the table', () => {})
+    test('can select a row on the resource table', async () => {
+        render(
+            <Router history={history}>
+                <ResourceIndex {...props} />
+            </Router>
+        )
+        await waitFor(() =>
+            fireEvent.click(screen.getAllByTestId('table-row')[0])
+        )
+    })
+    test('can search for values on the resource table', async () => {
+        render(
+            <Router history={history}>
+                <ResourceIndex {...props} />
+            </Router>
+        )
+        await waitFor(() =>
+            expect(screen.queryAllByTestId('table-row')).toHaveLength(2)
+        )
+        fireEvent.change(screen.getByTestId('search-resource'), {
+            target: { value: 'Amapai' },
+        })
+        window.Flamingo = {
+            ...FlamingoMock,
+            request: {
+                get: jest.fn().mockResolvedValue({
```

Comment:
Please try adding a test to make sure this function was called with the correct query parameter.

Reply:
Alright